pandas/io/formats/excel.py

```diff
@@ -381,12 +381,12 @@ def __init__(self, df, na_rep='', float_format=None, cols=None,
         self.inf_rep = inf_rep
 
     def _format_value(self, val):
-        if lib.checknull(val):
+        if libmissing.checknull(val):
             val = self.na_rep
         elif is_float(val):
-            if lib.isposinf_scalar(val):
+            if libmissing.isposinf_scalar(val):
```

Comment:
isposinf_scalar should be exposed in core/dtypes/missing.py as a python function